frontend/src/components/ui-kit/components/Select/Select.jsx

```diff
@@ -10,30 +10,35 @@ const Select = memo(({ value, setValue, options }) => {
     setValue(nextValue);
     setOpen(false);
   };
+  const handleMouseLeave = () => {
+    setOpen(false);
+  };
   return (
-    <div className={styles.root}>
-      <div className={styles.select} onClick={() => setOpen(!open)}>
-        <span className={styles.label}>
-          {options.find((option) => option.value === value)?.label}
-        </span>
-        {open ? <ICONS.arrowUp /> : <ICONS.showList />}
-      </div>
-      {open && (
-        <div className={styles.option}>
-          {options.map(
-            (option) =>
-              option.value !== value && (
-                <span
-                  key={option.value}
-                  className={styles.label}
-                  onClick={() => handleChange(option.value)}
-                >
-                  {option.label}
-                </span>
-              ),
-          )}
+    <div className={styles.container}>
+      <div className={styles.root} onMouseLeave={handleMouseLeave}>
```

Comment:
Why you need here onMouseLeave ?

Reply:
I use it to automatically close the select when the mouse leaves the field